mc-react-components/src/lib/date-range-picker/date-range-picker.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { shallow, mount, ReactWrapper } from 'enzyme';
+import { DateTimeRangePicker } from './date-range-picker';
+import { KeyboardDateTimePicker } from '@material-ui/pickers';
+import { Button } from '@material-ui/core';
+import { act } from '@testing-library/react';
+import { Wrapper } from '@material-ui/pickers/wrappers/Wrapper';
+
+const date = new Date(1990, 1, 1);
+
+it('set the values of to and from to the props of the date pickers when changed', () => {
+  const wrapper = mount(<DateTimeRangePicker onChange={() => {}} />);
+  wrapper.setProps({ from: date, to: date });
+
+  wrapper.update();
+  const pickers = wrapper.find(KeyboardDateTimePicker);
+  const fromRangePicker = pickers.first();
+  const toRangePicker = pickers.at(1);
+
+  expect(fromRangePicker.prop('value')?.valueOf()).toBe(date.getTime());
+  expect(toRangePicker.prop('value')?.valueOf()).toBe(date.getTime());
+});
+
+it('calls on change when button is clicked', () => {
+  const onChangeMock = jest.fn();
+  const wrapper = mount(
+    <DateTimeRangePicker from={date} to={date} onChange={onChangeMock} />
+  );
+  const button = wrapper.find(Button);
+
+  // @ts-ignore
```

Comment:
what is `@ts-ignore`? do we need it?

Reply:
Its used to suppress the typescript warning, so you can call onclick without supplying the parameters.